about.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+  <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+  />
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+  <link
+    rel="stylesheet"
+    href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    integrity="sha384-wvfXpqpZZVQGK6TAh5PVlGOfQNHSoD2xbE+QkPxCAFlNEevoEH3Sl0sibVcOQVnN"
+    crossorigin="anonymous"
+  />
+
+  <link rel="stylesheet" href="/factservices/styles.css" />
+  <body>
+    <header id="main">
+      <div class="header">
+        <div class="logo">
+          <img src="" alt="logo" />
+        </div>
+        <div class="cname">Fact Services</div>
+      </div>
+      <div class="nav">
+        <ul>
+          <li><a href="/factservices/main.html">Home</a></li>
+          <li><a href="/factservices/about.html">About</a></li>
+          <li><a href="#">Contact Us</a></li>
+        </ul>
+      </div>
+    </header>
+    <div class="about">
+      <div class="head"><h4>About Us</h4></div>
+      <p>
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eum ipsam,
+        beatae quis aspernatur, soluta, praesentium earum ad magni asperiores
+        rerum vero veritatis repellat officia dolorum harum doloremque fugit.
+        Eaque, facere.
```

Comment:
[factservices-43a43727c756d53119e306eccf7a03588d6c2650.zip](https://github.com/ogeobubu/factservices/files/5477173/factservices-43a43727c756d53119e306eccf7a03588d6c2650.zip)

Reply:
You've seen it right?